runtime/complex.go

```diff
@@ -140,3 +144,91 @@ func complexHash(f *Frame, o *Object) (*Object, *BaseException) {
 	}
 	return NewInt(hashCombined).ToObject(), nil
 }
+
+func complexArithmeticOpComplex(f *Frame, method string, v, w complex128, fun func(v, w complex128) complex128) (*Object, *BaseException) {
```

Comment:
Please move helper functions to the end of the file and keep method functions (e.g. complexAdd, complexSub) in alphabetical order below the initComplexType() function.